tooling/did-method-registry-entry.yml

```diff
@@ -49,5 +58,10 @@ example: {
     "contactName": "W3C Credentials Community Group",
     "contactEmail": "public-credentials@w3.org",
     "contactWebsite": "https://w3c-ccg.github.io/",
-    "verifiableDataRegistry": "None (ephemeral DID Method)"
+    "verifiableDataRegistry": "None (ephemeral DID Method)",
+    "implementation": [
+      "https://github.com/digitalbazaar/did-method-key-js",
+      "https://github.com/transmute-industries/did-key.js",
+      "https://github.com/decentralized-identity/did-key.rs"
```

Comment:
@clehner @troyronda you have changes you want to suggest here?

Reply:
If this were the actual did:key entry (key.json), I would have one to suggest (https://github.com/spruceid/ssi/tree/main/did-key). But as this is an example field I don't know if it's that useful... It would be an example of an implementation that's in a repo subdirectory rather than its own repo. Also it would make the list more balanced with two JavaScript and two Rust.

I wonder why "real" data is being used in this example field - Although did:key is exemplary in having many 
implementations, I would think it would be desired to avoid prioritizing/privileging a particular method and implementations in this documentation (key.json would easily get out of sync with this object) - but rather use e.g. `did:example` and `https://example.org/` links. But this PR follows the current approach so I think this question can be addressed separately/later.

Reply:
@OR13 suggested that we should use a "real example", we both thought did:key was a good non-controversial real-world example to use. Turns out the implementation field is controversial.

I'm not trying to advantage/disadvantage any implementation, just provide a reasonable example. I'm happy to list any did:key implementation in here that people want to list. Please put Spruce's entry in as a change suggestion and I'll add it, @clehner.

Reply:
Sorry I missed that conversation. Thanks for clarifying it. Change suggested: https://github.com/w3c/did-spec-registries/pull/378/files/bd5f4b9ae1bc1e1ebdf4f022e8f0ebbb0a5edd8a#r755429940